Resources/doc/definitions/resolver.md

```diff
@@ -11,16 +11,96 @@ Resolvers can be define 2 different ways
 
 1. **The PHP way**
 
-    You can declare resolver (any class that implements `Overblog\GraphQLBundle\Definition\Resolver\ResolverInterface`
+    You can declare a resolver (any class that implements `Overblog\GraphQLBundle\Definition\Resolver\ResolverInterface`
     or `Overblog\GraphQLBundle\Definition\Resolver\MutationInterface`)
-    in `src/*Bundle/GraphQL` or `app/GraphQL` they will be auto discover.
+    in `src/*Bundle/GraphQL` or `app/GraphQL` and they will be auto discovered.
     Auto map classes method are accessible by:
     * the class method name (example: `AppBunble\GraphQL\CustomResolver::myMethod`)
-    * the FQCN for callable classes (example: `AppBunble\GraphQL\InvokeResolver` for resolver implementing `__invoke` method)
-    you can also alias type implementing `Overblog\GraphQLBundle\Definition\Resolver\AliasedInterface`
-    that returns a map of method/alias. The service created will autowire `__construct`
+    * the FQCN for callable classes (example: `AppBunble\GraphQL\InvokeResolver` for a resolver implementing the `__invoke` method)
+    you can also alias a type by implementing `Overblog\GraphQLBundle\Definition\Resolver\AliasedInterface`
+    which returns a map of method/alias. The service created will autowire the `__construct`
     and `Symfony\Component\DependencyInjection\ContainerAwareInterface::setContainer` methods.
-    You can also define custom dirs using config:
+
+    Here an example:
+    ```php
+    <?php
+
+    namespace App\GraphQL\Resolver;
+
+    use Overblog\GraphQLBundle\Definition\Resolver\AliasedInterface;
+    use Overblog\GraphQLBundle\Definition\Resolver\ResolverInterface;
+
+    class Greetings implements ResolverInterface, AliasedInterface
+    {
+        public function sayHello($name)
+        {
+            return sprintf('hello %s!!!', $name);
+        }
+
+        /**
+         * {@inheritdoc}
+         */
+        public static function getAliases()
+        {
+            return ['sayHello' => 'say_hello'];
+        }
+    }
+    ```
+
+    `SayHello` resolver can be access by using `App\GraphQL\Resolver\Greetings::sayHello` or
```

Comment:
Maybe show how it looks on the yml `resolve`?
One per type would be good

Reply:
done